framework/arcane-framework/src/main/scala/services/storage/base/BlobStorageWriter.scala

```diff
@@ -0,0 +1,52 @@
+package com.sneaksanddata.arcane.framework
+package services.storage.base
+
+import services.storage.exceptions.BlobStorageException
```

Comment:
these are all models. So mb simply `main/scala/models/...`?

Reply:
Removed the exception at all